tinygrad/runtime/ops_metal.py

```diff
@@ -12,7 +12,7 @@
 class _METAL:
   def __init__(self):
     self.mtl_buffers_in_flight: List[Any] = []
-    self.device = Metal.MTLCreateSystemDefaultDevice()
+    self.device = [x for x in Metal.MTLCopyAllDevices() if x.name() not in getenv('CL_EXCLUDE', '').split(',')][0]
```

Comment:
Why was this added?

Reply:
Whoops - I will remove this and fix the linter issues.

Reply:
Apologies - I was testing other METAL code and needed to exclude the Intel GPU (defaults to it).  I've reverted this in the latest commit.